data/server_locations.json

```diff
@@ -39980,6 +39957,214 @@
                 "id": 6986,
                 "last_updated": "2023-11-30"
             }
+        },
+        {
+            "type": "Feature",
+            "geometry": {
+                "type": "Point",
+                "coordinates": [
+                    -98.1350768,
+                    26.0830897
+                ]
+            },
+            "properties": {
+                "name": "Santa Ana National Wildlife Refuge",
+                "active": false,
+                "area": "Texas",
+                "address": "3325 Green Jay Rd, Alamo",
+                "status": "retired",
+                "external_url": "http://209.221.138.252/Details.aspx?location=341637",
+                "internal_url": "null",
+                "latitude": "26.0830897",
+                "longitude": "-98.1350768",
+                "id": 3637,
+                "last_updated": "2023-12-04"
+            }
+        },
+        {
+            "type": "Feature",
+            "geometry": {
+                "type": "Point",
+                "coordinates": [
+                    -122.7581863,
+                    45.4347343
+                ]
+            },
+            "properties": {
+                "name": "Costco",
+                "active": true,
+                "area": "Oregon",
+                "address": "7850 SW Dartmouth St., Tigard",
+                "status": "unvisited",
+                "external_url": "http://209.221.138.252/Details.aspx?location=418535",
+                "internal_url": "null",
+                "latitude": "45.4347343",
+                "longitude": "-122.7581863",
+                "id": 6987,
+                "last_updated": "2023-12-05"
+            }
+        },
+        {
+            "type": "Feature",
+            "geometry": {
+                "type": "Point",
+                "coordinates": [
+                    -117.2194401,
+                    34.2397414
+                ]
+            },
+            "properties": {
+                "name": "Lake Arrowhead Tattoo",
+                "active": true,
+                "area": "California",
+                "address": "26744 CA-189, , Blue Jay",
+                "status": "unvisited",
+                "external_url": "http://209.221.138.252/Details.aspx?location=418466",
+                "internal_url": "null",
+                "latitude": "34.2397414",
+                "longitude": "-117.2194401",
+                "id": 6988,
+                "last_updated": "2023-12-05"
+            }
+        },
+        {
+            "type": "Feature",
+            "geometry": {
+                "type": "Point",
+                "coordinates": [
+                    -86.9844129,
+                    30.40116159999999
+                ]
+            },
+            "properties": {
+                "name": "Gulf Breeze Zoo",
```

Comment:
Here the Gulf breeze Zoo appears again - before it was deleted in the server_locations and added in the all locations; and it is shown as retired here, although there was a user comment that it's only broken..? By the way, I think we can start setting "active" = "out-of-order", so that we can save this information. So far we don't use the "active" attribute, so it should not cause any problems to change it to a string

Reply:
Aaah super well spotted with the Breeze machine, that one should certainly be active!! 🚀